mhkit/wave/io/cdip.py

```diff
@@ -0,0 +1,151 @@
+import pandas as pd
+import numpy as np
+import netCDF4
+import datetime
+import time
+
+def request_data(stn,start_date='',end_date='',yeardate='',data_type='Historic'):
+    """
+    Requests CDIP data by wave buoy data file (from http://cdip.ucsd.edu/).
+    
+
+    Parameters
+    ------------
+    stn: string
+        Station number of CDIP wave buoy
+    start_date: string
+        Start date in MM/DD/YYYY, e.g. '04/01/2012'
+    end_date: string
+        End date in MM/DD/YYYY, e.g. '04/30/2012'
+    yeardate: string
+        Year date, e.g. '2001'
+    data_type: string
+        'Realtime' or 'Historic', default = 'Historic'
+    
+    Returns
+    ---------
+    data: pandas DataFrame 
+        Data indexed by datetime with columns named according to the data 
+        signal, for it includes: Hs, Tp, and Dp
+        
```

Comment:
buoy name is also returned. I would suggest we move this into the Pandas DataFrame name or column name. E.g, `data.name = buoyname`.

Reply:
@ssolson 

- Did you see my note "Unable to [configure proxy for netcdf4](https://github.com/Unidata/netcdf4-python/search?q=proxy&type=issues)"? I also get a timeout error when I'm on the VPN. Does this resolve your issue?
- I plan to write tests for this functionality on the future, it just hasn't been done yet. My intent was to request a preliminary review on the functionality/implementation prior to completing it. But, if tests will help debug the issue, I can prioritize developing them, or you can :)
- I'll add the the ability to store the buoy name, that's a good idea.